validate.js

```diff
@@ -11,6 +11,38 @@ const exportedMethods = {
     return id;
   },
 
+  checkStevensMail(emailid) {
+    if (!emailid) throw new Error(`Expected Emailid to be non-empty`);
+    if (typeof emailid !== "string" || emailid.trim().length === 0)
+      throw new Error(`Expected Emailid to be non-empty string`);
+    emailid = emailid.trim().toLowerCase();
+    let regex = /^[\w._%+-]+(@stevens\.edu)$/;
+    if (!regex.test(emailid))
+      throw new Error(`Expected email id to be of Stevens`);
+    return emailid;
+  },
+
```

Comment:
Can you check If entered stevens email id is a registered Id and Not some random stevens id?

Reply:
Yep only pthakur2@stevens.edu will go through might add additional check for pthakur2 as well later